meta-xt-domd-gen3/recipes-core/images/core-image-weston.bbappend

```diff
@@ -1,4 +1,5 @@
 IMAGE_INSTALL:append = "\
-    ${@bb.utils.contains('DISTRO_FEATURES', 'enable_virtio', ' virglrenderer libsdl2 qemu-system-aarch64 qemu-keymaps', '', d)} \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'enable_virtio wayland', ' virglrenderer libsdl2', '', d)} \
```

Comment:
Where is `enable_virtio_wayland` is added to the DISTRO_FEATURES? I can't see relevant changes to the prod-devel-rcar-virtio.yaml.

Reply:
This is not `enable_virtio_wayland` but two separate DISTRO_FEATURES `enable_virtio` and `wayland`.
- `enable_virtio` is defined inside virtio.yaml for Dom0 and DomD
- `wayland` is defined inside `${POKY_DEFAULT_DISTRO_FEATURES}` in `poky/meta-poky/conf/distro/poky.conf`

Function `bb.utils.contains` allows to specify more than one value separated by spaces and `Check if a variable contains all the values specified`.

So here we have condition "If DISTRO_FEATURES contains `enable_virtio` AND `wayland`"

Reply:
Right, my bad.